src/snoowrap.js

```diff
@@ -769,6 +769,29 @@ const snoowrap = class snoowrap {
   getNewComments (subredditName, options) {
     return this._getSortedFrontpage('comments', subredditName, options);
   }
+  
+  
+  /**
+   *  @summary Get list of content by IDs.
+   *  @param {Array<string|Submission|Comment>} ids An array of fullname IDs you want to fetch the content of.
+   *  @returns {Promise} A listing of content requested, can be any class fetchable by API. e.g. Comment, Submission
+  */
+  getContentByIds (ids) {
+    if (typeof ids[0] === 'object') {
+      if ('name' in ids[0]) {
+        const objectId = [];
+        for (let i = 0; i < ids.length; i++) {
+          objectId.push(ids[i].name);
+        }
+        ids = objectId;
+      }
+    }
```

Comment:
What happens if I send in an empty array, or a mixed array of strings and objects? Instead, replace this with a loop

From the top of my head I could see it as this, though most likely a better way to do it

```js
const prefixedIds = ids.map(id => {
  if (id instanceof snoowrap.objects.Submission || id instanceof snoowrap.objects.Comment) {
    return id.name;
  }
  else if (typeof id === "string") {
    if (!/t(1|3)_/g.test(ids)) {
      throw new TypeError('Invalid argument: List of IDs need to be fullnames.');
    }
    return id;
  }
  else {
    throw new TypeError("Id must be either a string, Submission, or Comment.")
  }
});
```

Reply:
Didn't even originally think about using map! That'd work so much better!